LL-4147 fix(accounts): make account row fully clickable

jgiamporcaro-ledger opened the pull request:
Account list padding was not clickable. From now the row will be fully clickable, padding included.

https://user-images.githubusercontent.com/33158502/124952643-b5dc1800-e014-11eb-82ca-41ef1da0bc08.mov

<!-- Description of what the PR does go here... screenshot might be good if appropriate -->

### Type

<!-- e.g. Bug Fix, Feature, Code Quality Improvement, UI Polish... -->

Fix

### Context

[LL-4147]

### Parts of the app affected / Test plan

<!-- Which part of the app is affected? What to do to test it, any special thing to do? -->

Rows from the account list.

[LL-4147]: https://ledgerhq.atlassian.net/browse/LL-4147

LFBarreto reviewed (dismissed):
could you test if the right click context menu is still working properly with those changes? :+1:

juan-cortes commented:
> could you test if the right click context menu is still working properly with those changes? 👍

I think you're right, if the left click was impacted, the right click is impacted too, probably move the `AccountContextMenu` up one level or something along those lines.

jgiamporcaro-ledger commented:
NIce catch mates, thanks!

I moved up the `AccountContextMenu` component and fixed up the change to my initial [commit](https://github.com/LedgerHQ/ledger-live-desktop/pull/3970/commits/5833b8ad9ab6681504caa04b1754f873aa9f9d49)

https://user-images.githubusercontent.com/33158502/125078418-a61b0d00-e0c2-11eb-948e-83e0c0899c22.mov

LFBarreto reviewed (approved):
:+1:

nabil-brn reviewed (changes requested):
Unfortunately _left-click_ is no more working on sub accounts, I'm redirected to the parent account.

---
2.30.0-sha.5833b8ad9ab6681504caa04b1754f873aa9f9d49-win.exe.zip

jgiamporcaro-ledger commented:
Yeah, my bad, I didn't have ETH tokens so subAccounts wasn't listed when I fix the bug.
Thanks for sending me some tokens yesterday, I'll take a look on this one today 🦾

jgiamporcaro-ledger commented:
@nabil-brn 

Fixed 🔥 

https://user-images.githubusercontent.com/33158502/125957878-6f78b40a-49fb-442d-81eb-77e1211a71fa.mov

jgiamporcaro-ledger commented on src/renderer/screens/accounts/AccountRowItem/index.js:
```diff
@@ -84,7 +87,8 @@ const TokenBarIndicator: ThemedComponent<{}> = styled.div`
   position: absolute;
   left: 0;
-  height: 100%;
+  margin-top: -20px;
+  height: calc(100% + 20px);
```
These lines stick the vertical line to content title.

nabil-brn reviewed (approved):
:heavy_check_mark:  QA-ed

---
ledger-live-desktop-2.31.0-sha.a2a9b289642db9ceb38be7acd20195acd6bccd96-linux-x86_64.AppImage
